mod_form.php

```diff
@@ -122,9 +122,13 @@ public function definition() {
         $mform->setType('confirmationsubject', PARAM_TEXT);
         $mform->setDefault('confirmationsubject', get_string('setting:defaultconfirmationsubjectdefault', 'facetoface'));
 
-        $mform->addElement('textarea', 'confirmationmessage', get_string('email:message', 'facetoface'), 'wrap="virtual" rows="15" cols="70"');
-        $mform->setDefault('confirmationmessage', get_string('setting:defaultconfirmationmessagedefault', 'facetoface'));
-
+        $editoroptions = ['trusttext' => true];
+        $mform->addElement('editor', 'confirmationmessage', get_string('email:message', 'facetoface'), null, $editoroptions);
+        $mform->setType('confirmationmessage', PARAM_RAW);
+        $confirmationmessagedata = [
+            'text' => get_string('setting:defaultconfirmationmessagedefault2', 'facetoface'),
+            'format' => 1
```

Comment:
Magic number? "1" - shouldn't this be FORMAT_HTML ?

Reply:
Fixed